bundler/lib/dependabot/bundler/helpers.rb

```diff
@@ -11,6 +11,14 @@ module Helpers
       def self.bundler_version(_lockfile)
         V1
       end
+
+      # TODO: Replace with bundler_version once it is implemented
+      def self.actual_bundler_version(lockfile)
+        return V1 unless lockfile
```

Comment:
Should we return `unknown` here?

I guess the question ultimately is: should we instrument the _version used_ or the _version specified_

Reply:
Ah yes, I think it would be useful to gauge the usage of current users so returning `V1` makes sense to me 👍

Reply:
Hm I just re-read this, good point when there's no lockfile, I guess we'll eventually default to `V2` here so maybe a good idea to return `uknown` actually so we can get a sense for how many projects would be impacted by flipping the default, thoughts?

Reply:
Yeah, agreed 👍 we'll probably want to keep this separate helper so I'll think of a more permanent name